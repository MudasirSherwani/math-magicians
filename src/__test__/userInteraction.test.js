import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Navbar from '../components/Screen';

describe('navbar component', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(<BrowserRouter><Navbar /></BrowserRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
