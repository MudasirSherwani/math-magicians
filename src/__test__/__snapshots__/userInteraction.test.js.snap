// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navbar component should render correctly 1`] = `
<div
  className="screen"
>
  0
</div>
`;
